src/utils/github-data-parser.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * takes an object from github data from
+ * GET /users/{username}/repos
+ * and rips out only relevant keys
+ * @param {Array} userRepoData
+ *
+ */
+export const parseUserData = (userData) => {
```

Comment:
nit - consider something like "selectUserDataForDisplay" and same for below (not huge deal)

Reply:
hmm not the biggest fan of that name. we do want to select certain keys from an object but i think when the right words come to mind we can change it